templates/App/_partials/_nav/_shared.html.twig

```diff
@@ -14,6 +14,24 @@
     </form>
 </li>
 
+<li>
+<div class="locale-dropdown-sidebar" data-controller="dropdown">
```

Comment:
This whole section needs to be tabbed?

Please consider

Reply:
You're right, fixed